src/portable/st/synopsys/dcd_synopsys.c

```diff
@@ -356,8 +354,6 @@ bool dcd_edpt_xfer (uint8_t rhport, uint8_t ep_addr, uint8_t * buffer, uint16_t
   xfer_ctl_t * xfer = XFER_CTL_BASE(epnum, dir);
   xfer->buffer       = buffer;
   xfer->total_len    = total_bytes;
-  xfer->queued_len   = 0;
-  xfer->short_packet = false;
 
   uint16_t num_packets = (total_bytes / xfer->max_size);
```

Comment:
it is not important, but it is good practice to put `const` on all fixed variable. I made to difference to the code, but give a hint on how we want to use the variable in following code.